src/test/java/uk/gov/cshr/AppInsightsHelper/Helper.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.cshr.AppInsightsHelper;
+
+import com.microsoft.applicationinsights.web.internal.WebRequestTrackingFilter;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.springframework.stereotype.Component;
+import org.springframework.test.web.servlet.MockMvc;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletContext;
+import java.util.Enumeration;
+
+@Component
+public class Helper {
```

Comment:
just a thought. `FilterHelper` can have better meaning than just `Helper`. Nice to see it is annotated as a spring bean though.

Reply:
yip